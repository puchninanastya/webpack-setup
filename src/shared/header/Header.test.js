import React from 'react';
import renderer from 'react-test-renderer';

import Header from './index';

it('Header renders correctly with custom title', () => {
  const header = renderer
    .create(<Header title="Test title" />)
    .toJSON();
  expect(header).toMatchSnapshot();
});

it('Header renders correctly with default title', () => {
  const header = renderer
    .create(<Header />)
    .toJSON();
  expect(header).toMatchSnapshot();
});