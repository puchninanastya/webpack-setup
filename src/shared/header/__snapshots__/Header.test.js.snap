// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly with custom title 1`] = `
<h1
  className="header"
>
  Test title
</h1>
`;

exports[`Header renders correctly with default title 1`] = `
<h1
  className="header"
>
  Hi there
</h1>
`;
